I've added my functions and my code

tarikthomas opened the pull request:
I've added all my functions and my code to the dashboard, I'm not sure how GitHub will merge the files.

tarikthomas committed:
I've added my functions and my code

I've added all my functions and my code to the dashboard, feel free to look over it

yeewaifatt reviewed (commented):
Changes reviewed and approved.

tarikthomas committed:
Tarik - Attempt to Merge my code into Dashboard

I have downloaded the files (dashboard and dashboard_functions) from the main branch and added my coded to it so that nothing conflicts, hopefully.
